src/language/classes.md

```diff
@@ -167,19 +167,16 @@ class Point {
 }
 ```
 
-All uninitialized instance variables have the value `null`.
+Uninitialized instance variables declared with
+[nullable types][] have the value `null`.
```

Comment:
I usually try to minimize the use of plural in situations like this, because of the tendency to introduce ambiguities. This could also be a matter of style/convention (it would be a bad idea to switch to singular if this location would then stand out as an exception). Otherwise:

```suggestion
An uninitialized instance variable declared with a
[nullable type][] has the value `null`.
```

Reply:
That's a great point, thank you!